IBM/openapi-validator
# refactor: use Spectral validation for unique operationId

Changes:
- enable the `operation-operationId-unique` Spectral rule
- remove in-code validation that checks for duplicate operationIds

Notable results:
- The error message updated to say, 'Every operation must have a unique `operationId`.'
- The error is presented for all locations in which the duplicate opId is used. Line numbers provided for all occurrences, which is an enhancement.

Tests:
- Remove tests for in-code checks for unique operationIds
- Update tests to account for the additional errors reported (one error for each all instances of a duplicated operationId instead of one error per duplicate operationId)
- Update tests to reflect changing order of output
- Update tests to reflect change in expected output (messages, line numbers, etc.)
